app/assets/stylesheets/grader.scss

```diff
@@ -184,9 +184,8 @@
 
 .mark-deductions {
   clear: both;
-  color: $dark-blue;
-  cursor: pointer;
```

Comment:
Just double-checking this: does the `cursor: pointer` still apply for the mark deductions themselves (e.g., "-1")? Because those are clickable.

Reply:
Yep!